src/main/java/com/sixt/service/framework/servicetest/service/ServiceUnderTestImpl.java

```diff
@@ -146,6 +146,17 @@ public String sendHttpPost(String path, String data) throws Exception {
     }
 
     @Override
+    public <TYPE extends Message> List<TYPE> getEventsOfType(Class<TYPE> eventClass) {
+
+        if (eventHandler == null) {
+            logger.warn("Event handler has not been initialized. Create the " +
```

Comment:
i didn't write the event-handling in the SIT area.  if you change both to throw exceptions, i'll merge it ;)  i don't like warnings like this because i have spent too much time trying to debug tests in depth just to find there was a warning in the logs